package.json

```diff
@@ -72,6 +69,7 @@
     "eslint-plugin-promise": "^5.1.0",
     "eslint-plugin-react": "^7.24.0",
     "eslint-plugin-unicorn": "^42.0.0",
+    "hardhat": "^2.10.2",
```

Comment:
It doesn't seem to be used in the home directory

Reply:
This project use lernajs, hardhat for local make up,  eslint is for ts grammar check.  when run make will exec package dir ts code. So it need

Reply:
hardhat looks unnecessary in the home directory and can be removed